crates/loader/src/local/mod.rs

```diff
@@ -93,6 +98,37 @@ fn error_on_duplicate_ids(components: Vec<RawComponentManifest>) -> Result<()> {
     Ok(())
 }
 
+fn validate_raw_app_manifest(raw: &RawAppManifestAnyVersion) -> Result<()> {
+    match raw {
+        RawAppManifestAnyVersion::V1(raw) => {
+            let _ = raw
+                .components
+                .iter()
+                .map(|c| validate_raw_wasm_config(&c.wasm))
+                .collect::<Result<Vec<_>>>()?;
+        }
+    }
+    Ok(())
+}
+
+fn validate_raw_wasm_config(raw: &RawWasmConfig) -> Result<()> {
```

Comment:
This is the same function as in the bindle loader.  I know the `RawWasmConfig` types are different between the two, but the `allowed_http_hosts` types are the same.  Could we have a helper function that takes `&Option<Vec<String>>` (or similar) and has all the logic in it, and call that function from both places?

Reply:
Thanks for your suggestion. I add a `validate_allowed_http_hosts` function and remove `validate_raw_wasm_config` in both `local` and `bindle` loader.